src/styles/theme.js

```diff
@@ -147,6 +147,10 @@ export default ({ color, backgroundColor, borderColor, cardColor }: Props) => ({
   heading2: {
     fontSize: 20,
   },
+  headingLarge: {
+    fontSize: 40,
+    color: 'rgb(100, 100, 100)',
```

Comment:
Hard coding the color this way would break the screen in night mode, you should preferably use default color for the current theme. This way the view won't break in night mode also.

Reply:
I updated the commit, now it uses the default color.